core/sql/sqlcomp/CmpSeabaseDDLrepos.cpp

```diff
@@ -47,8 +47,17 @@ short CmpSeabaseDDL::createRepos(ExeCliInterface * cliInterface)
 
   NABoolean xnWasStartedHere = FALSE;
 
+  cliRC = cliInterface->holdAndSetCQD
+    ("TRAF_MAX_CHARACTER_COL_LENGTH", "1000000");
+
```

Comment:
The EXPLAIN_PLAN column in the METRIC_QUERY_TABLE was increased in size from 200000 to 1000000 as part of these changes.